charts/rancher-cis-benchmark/4.0.0-rc4/templates/rbac.yaml

```diff
@@ -62,10 +71,45 @@ rules:
   - "namespaces"
   - "nodes"
   - "pods"
+  - "serviceaccounts"
+  - "services"
+  - "replicationcontrollers"
   verbs:
   - "get"
   - "list"
   - "watch"
+  - "patch"
```

Comment:
Is the `patch` verb needed?

Reply:
Ah! right. No we don't need it. Removed and updated the PR.